tests/pass-dep/shims/libc-fs-with-isolation.rs

```diff
@@ -1,4 +1,5 @@
 //@ignore-target-windows: no libc on Windows
+//@ignore-target-freebsd
```

Comment:
```suggestion
//@ignore-target-freebsd: FIXME needs foreign function `stat@FBSD_1.0`
```

Reply:
Note could be fixed in "all fs" next PR eventually.